src/sso/pages/AppDetails.tsx

```diff
@@ -65,6 +78,17 @@ const AppDetails = () => {
     setDetails(app);
   }
 
+  const logout = async () => {
+    if (user) await user.logout();
+    // window.location.href = document.referrer;
+    const response = {
+      success: false,
+      reason: SdkErrors.UserLogout,
+    };
+
```

Comment:
why is the reason an error ? or an SDK error ?

Reply:
hi @theblockstalk suggested to add error enum and use it here

Reply:
correct. this is good enough for now